### ISSUE TITLE
Does FromSampleJson support draft6 or draft7?Do you have any plan to support?

### ISSUE THREAD

**Issue by xiaowu351** [RSuter/NJsonSchema#912]
hello, RSuter:
Does FromSampleJson support draft6 or draft7?Do you have any plan to support? Thanks

**Comment by RSuter** [comment]
What is missing? FromSampleJson is very basic anyway and probably does not need any draft6/7 features...

**Comment by xiaowu351** [comment]
Currently, FromSampleJson outputs draft4 format instead of draft6/7.I wanted to export the json schema for draft6/7 based on json data.

**Comment by RSuter** [comment]
So it's only the $schema which is "wrong"?

**Comment by xiaowu351** [comment]
No, This generates fewer Json Schema elements than draft6/7. Such as the absence of "required"

**Comment by RSuter** [comment]
Probably the generation of required is just not implemented yet, the from json sample generator is quite basic at this time...

**Comment by xiaowu351** [comment]
Oh, Future version plan implement or keep it the same?
